kotlin/hexagon-netty-async/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 ext {
-    hexagonVersion = "3.0.0-B9"
```

Comment:
is this something we can remove @jaguililla ? seems that hexagon is 3.3, but this adapter does not exists